.github/workflows/release.yml

```diff
@@ -50,3 +50,5 @@ jobs:
         run: |
           tag="$(echo "${{ github.ref }}" | tr -d 'refs/tags/v')"
           docker buildx build --build-arg copa_version=${tag} -t ghcr.io/project-copacetic/copa-action:v"$tag" --push .
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
you'll need docker login step first

even better, can we integrate this to goreleaser? see https://goreleaser.com/customization/docker/?h=docker